context/app/static/js/components/Detail/FilesConditionalLink/style.js

```diff
@@ -1,8 +1,8 @@
 import styled from 'styled-components';
 import { LightBlueLink } from 'shared-styles/Links';
 
-const StyledLink = styled(LightBlueLink)`
-  cursor: pointer;
+const AlignedLink = styled(LightBlueLink)`
+  vertical-align: baseline;
```

Comment:
I can't recall now why `pointer` was added, but if it's no longer needed, great!

Reply:
You needed pointer at the time because the element was an anchor without an href.